jokes.json

```diff
@@ -22,5 +22,9 @@
   {
     "joke": "Você conhece a piada do fotógrafo? Ainda não foi revelada",
     "author": "Desconhecido"
+  },
+  {
+    "joke": "É claro que a terra é plana! Olha o nome, Planeta! Se fosse redonta era Redondeta!",
```

Comment:
What is "redonta"?

Reply:
It means: "pau no cú dos professores de gramática do repositório"
The issue was revised and corrected

Reply:
What is "cú" with an accented character?

Reply:
it means: "brioco"

Reply:
🤣